USER_GUIDE.md

```diff
@@ -0,0 +1,206 @@
+# User Guide: Running HuggingFace LLaMA 2 Training on v4 and v5e
+
+
+This user guide provides a concise overview of the essential steps required to run HuggingFace (HF) Llama 2 training on both v4 and v5e. It first demonstrates how to do so on barebone TPU VMs, and then shares a Dockerfile for those who would prefer to run it in a container.
+
+
+## Environment Setup
+
+The following setup assumes to run the training job with Llama 2 7B.
+
+
+### Cloud TPU VM Creation
+```
+export TPU_NAME=jwtan-v4-32
+export ZONE=us-central2-b
+export ACCELERATOR_TYPE=v4-32
+export RUNTIME_VERSION=tpu-ubuntu2204-base
+export PROJECT=mlperf-high-priority-project
+
+gcloud alpha compute tpus queued-resources create $TPU_NAME --node-id $TPU_NAME --zone=$ZONE --project=$PROJECT --accelerator-type=$ACCELERATOR_TYPE --runtime-version=$RUNTIME_VERSION
```

Comment:
If we split this into a few lines, there won't be a scrollbar when rendered

Reply:
Not sure what do you mean? Mind a screenshot. Here is my rendered version:
<img width="966" alt="Screenshot 2023-09-26 at 3 38 18 PM" src="https://github.com/pytorch-tpu/transformers/assets/8573935/a4a39838-a1df-44e5-9350-599bc0f45eab">

Reply:
From https://github.com/pytorch-tpu/transformers/blob/933a18a56a612e37009ef5507bfa5bce2173fd9d/USER_GUIDE.md

![image](https://github.com/pytorch-tpu/transformers/assets/14892068/46c2918d-4a26-4d6f-8d0e-daa106d9e555)

Reply:
Let me chop it. vscode is too smart.